examples/todoapp/lib/feature_todo_management/blocs/todo_management_bloc.dart

```diff
@@ -114,6 +114,7 @@ class TodoManagementBloc extends $TodoManagementBloc {
         _todoSubject,
         (title, todo) => title ?? todo.title,
       )
+          .skip(1)
```

Comment:
Why we need to skip the first event ?

Reply:
It always emits an empty string at the beginning

Reply:
Please leave a comment in this case